x-pack/filebeat/fbreceiver/receiver.go

```diff
@@ -37,6 +47,33 @@ func (fb *filebeatReceiver) Start(ctx context.Context, host component.Host) erro
 func (fb *filebeatReceiver) Shutdown(ctx context.Context) error {
 	fb.logger.Info("stopping filebeat receiver")
 	fb.beater.Stop()
+	if err := fb.stopMonitoring(); err != nil {
+		return fmt.Errorf("error stopping monitoring server: %w", err)
+	}
 	fb.wg.Wait()
 	return nil
 }
+
+func (fb *filebeatReceiver) startMonitoring() error {
+	if fb.httpConf.Enabled() {
+		var err error
+		fb.beat.RegisterMetrics()
+		err = metricreport.SetupMetrics(logp.NewLogger("metrics"), fb.beat.Info.Beat, version.GetDefaultVersion())
+		if err != nil {
+			return err
+		}
+		fb.beat.API, err = api.NewWithDefaultRoutes(logp.NewLogger(""), fb.httpConf, monitoring.GetNamespace)
```

Comment:
I believe it would be nice to name the logger here. May be `metrics` or `metrics.http`

Does `api.NewWithDefaultRoutes` name the logger?